build.xml

```diff
@@ -1148,9 +1148,26 @@ ext = BeanUtils.cook(ext);
         <unzip dest='${webapp.dir}' src="${webapp.file}"/>
 
         <antcall target='webxml-deploy-replace'/>
+        <antcall target='timezones-deploy'/>
```

Comment:
Can we confirm if this change doesn't affect the change done by @shri314  (https://github.com/Zimbra/zm-web-client/pull/110)  ? 
The jam-files target was changed there. With this change in 'deploy' , internally 'jam-files' too get  called. 
Can we confirm if the SE team is not using this target at all and just depending on the war generated ?